README.md

```diff
@@ -136,267 +132,43 @@ For an even more in-depth demo, check out our [Deploying to Kubernetes](https://
-
-Example:
-```
-...
-    credentials:
-      accessToken:
-        from: kubernetes
-        get: secret_identifier#key
-...
-```
-
-### File Provider
-
-File (`file`) provider allows you to use a file available to the Secretless Broker process and/or container as sources of
-credentials.
-
-Example:
-```
-...
-    credentials:
-      rsa:
-        from: file
-        get: /path/to/file
-...
-```
-
-### Environment Variable Provider
-
-Environment (`env`) provider allows use of environment variables as source of credentials.
-
-Example:
-```
-...
-    credentials:
-      accessToken:
-        from: env
-        get: ACCESS_TOKEN
-...
-```
-
-### Literal Value Provider
-
-Literal (`literal`) provider allows use of hard-coded values as credential sources.
-
-_Note: This type of secrets inclusion is highly likely to be much less secure versus other
-available providers so please use care when choosing this as your secrets source._
-
-Example:
-```
-...
-    credentials:
-      accessToken: "supersecretaccesstoken"
-...
-```
-
-### Keychain Provider (beta)
-
-Keychain (`keychain`) provider allows use of your OS-level keychain as the credentials provider.
-
-_Note: This provider currently only works on Mac OS at the time and only when building from source so it should
-be avoided unless you are a developer working on the source code. There are plans to integrate all major OS
-keychains into this provider in a future release._
-
-Example:
-```
-...
-    credentials:
-      rsa:
-        from: keychain
-        get: servicename#accountname
-...
-```
-
-## Health Checks
-
-Secretless broker exposes two endpoints that can be used for readiness and liveliness checks:
-
-- `http://<host>:5335/ready` which will indicate if the broker has loaded a valid configuration.
-- `http://<host>:5335/live` which will indicate if the broker has service connectors activated.
-
-If there are failures, the service will return a `503` status or a `200` if the service indicates that
-the broker is ready/live.
-
-Note: If Secretless is not provided with a configuration (e.g. it is not listening to anything),
-the live endpoint will also return 503.
-
-You can manually check the status with these endpoints by using `curl`:
-```
-$ # Start Secretless Broker in another terminal on the same machine
-
-$ curl -i localhost:5335/ready
-HTTP/1.1 200 OK
-Content-Type: application/json; charset=utf-8
-Date: Thu, 27 Dec 2018 17:12:07 GMT
-Content-Length: 3
-
-{}
-```
-
-If you would like to retrieve the full informational JSON that includes details on
-which checks failed and with what error, you can add the `?full=1` query parameter
-to the end of either of the available endpoints:
-```
-$ # Start Secretless Broker in another terminal on the same machine
-
-$ curl -i localhost:5335/ready?full=1
-HTTP/1.1 200 OK
-Content-Type: application/json; charset=utf-8
-Date: Thu, 27 Dec 2018 17:13:22 GMT
-Content-Length: 45
-
-{
-    "listening": "OK",
-    "ready": "OK"
-}
-```
```

Comment:
I don't agree that we should be removing this content. While we do have that content on our documentation website, it will probably prevent many developers from adopting our product since you can't just take a look at the README itself to get an idea on how to use/run this thing.

Reply:
our README is so long, even with this removed. looking at other repos, eg [kubernetes](https://github.com/kubernetes/kubernetes/), [envoy](https://github.com/envoyproxy/envoy/), [linkerd](https://github.com/linkerd/linkerd2), there has been a movement in this space to make the readme as simple as possible and refer out to other docs. there are also major benefits in terms of keeping things up-to-date of having a single source-of-truth for any given piece of data.

I just don't think there's a real benefit from having this info in the README anymore.